styles/styles.css

```diff
@@ -281,3 +282,69 @@ body {
   font-size: 12px;
   margin: 20px 0 72px 0;
 }
+
+.contact-form {
```

Comment:
- [ ] As Figma design present a background image for the contact form, please implement that for your project too.
Figma design:
![image](https://user-images.githubusercontent.com/83596135/160712362-8f7cf053-c21c-47c0-bfd4-5a34bf89f63b.png)